cookbooks/bcpc/templates/default/cobbler/select_bach_root_disk.erb

```diff
@@ -39,7 +39,7 @@ udevadm settle --timeout=30
   end
 
   candidate_disks_glob = driver_globs.join(' ') +
-    '/dev/sd[a-z] /dev/sd[a-z][a-z]'
+    ' /dev/sd[a-z] /dev/sd[a-z][a-z] /dev/nvme*[0-9]n*[0-9]'
```

Comment:
Prepended a space to this string to prevent `/dev/sd[a-z]` from being appended to the last entry in the `driver_globs` array like `/dev/ahci/*/dev/sd[a-z]`

Reply:
Also needed to use `*[0-9]` rather than `[0-9]+` here, since the latter is invalid in `ash`.